packages/wrangler/src/dev/local.tsx

```diff
@@ -295,10 +296,18 @@ function useLocalWorker({
 				return;
 			}
 
+			// Wait for the Worker port to be available. We don't want to do this in experimental local
+			// mode, as we only `dispose()` the Miniflare 3 instance, and shutdown the server when
+			// unmounting the component, not when props change. If we did, we'd just timeout every time.
+			await waitForPortToBeAvailable(initialPort, {
+				retryPeriod: 200,
+				timeout: 2000,
+				abortSignal: abortController.signal,
+			});
+
 			const nodeOptions = setupNodeOptions({
 				inspect,
-				ip: initialIp,
```

Comment:
Why is the ip argument being removed?

Reply:
Does that remove the ability for user defined IP, or did it not have that capability before?